gems/sorbet/lib/user_input.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Sorbet::Private::UserInput
+  def prompt_for_confirmation!(prompt)
```

Comment:
Instead of extending can you please just make this a static method?

Reply:
Yep!